tests/Unit/Emogrifier/CssConcatenatorTest.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Pelago\Tests\Unit\Emogrifier;
+
+use Pelago\Emogrifier\CssConcatenator;
+
+/**
+ * Test case.
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ */
+class CssConcatenatorTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var CssConcatenator
+     */
+    private $subject = null;
+
+    /**
+     * @return void
+     */
+    protected function setUp()
+    {
+        $this->subject = new CssConcatenator();
+    }
+
+    /**
+     * @test
+     */
+    public function getCssInitiallyReturnsEmptyString()
+    {
+        $result = $this->subject->getCss();
+
+        static::assertSame('', $result);
+    }
+
+    /**
+     * @test
+     */
+    public function getCssReturnsSingleAppendedRule()
+    {
+        $this->subject->append('p', 'color: green;');
+
+        $result = $this->subject->getCss();
+
+        static::assertSame('p{color: green;}', $result);
+    }
+
+    /**
+     * @test
+     */
+    public function getCssReturnsSingleAppendedMediaRule()
```

Comment:
Accordingly, in this case the name then would be `appendWithMediaRuleAppendsToMediaRule` (or something similar).

Reply:
Renamed 2 test methods.